code/modules/food_and_drinks/food/foods/candy.dm

```diff
@@ -644,3 +644,4 @@
 	icon_state = "toolerone"
 	filling_color = "#7D5F46"
 	goal_difficulty = FOOD_GOAL_NORMAL
+
```

Comment:
```suggestion
```
Just forgot to remove that empty line.
Will be reviewing this more once the idea is approved by the team.